collect_app/src/main/AndroidManifest.xml

```diff
@@ -98,7 +98,11 @@ the specific language governing permissions and limitations under the License.
         </provider>
 
         <activity
-            android:name=".activities.FirstLaunchActivity"
+            android:name=".activities.FirstLaunchActivity">
+        </activity>
+
+        <activity
+            android:name=".activities.MainMenuActivity"
```

Comment:
Switching to the `MainMenuActivity` is a pretty big change. I needed to make that happen to account for the Android's ["restart on crash" behaviour](https://www.seadowg.com/2022/10/24/crash-test.html).

Reply:
An Activity like that would need to either `finish()` when launching the next Activity or have `noHistory` which is what causes the problem when the app restarts (for a crash or because of a memory reclaim): the Activity is not in the backstack that's restored.